src/app/ad-authoring/ad-authoring.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="ad-authoring">
+    <br>
+    <mat-expansion-panel>
+        <mat-expansion-panel-header>
+            <mat-panel-title>
+                Text
+            </mat-panel-title>
+        </mat-expansion-panel-header>
+
+        <mat-form-field>
+            <mat-label>Landing Page URL</mat-label>
+            <input matInput 
+                type="text" 
+                #landingUrl 
+                placeholder="Landing Page URL" 
+                [ngModel] = "landingPageUrl"
```

Comment:
Can you keep the naming consistent as landingPageUrl or landingUrl across the code?